README.md

```diff
@@ -2,72 +2,86 @@
+- A migration.  This creates a table in the database for forum records.
 - A model. This is the Active Record class for the forum records.  Active Record is an object relational mapper, which converts operations on Active Record objects into SQL.
 - A route is added.
 - A controller is created.
 - Views are created.
 - And some other stuff as well, having to do with testing and the like -- you can ignore this.
 
-So, let's look at each.  
+So, let's look at each.
 
-- First, the migration file.  It is in the db/migrate directory.  The name is a timestamp, plus a descriptive name saying what it does.  Open it up.  You'll see that it creates the table in the database.
-- Second, the model.  This is app/models/Forum.rb.  If you have a look, you'll see that there isn't any code there.  The model gets all of its behavior at this point from the parent class and the schema in the database.  We'll modify model files in a later lesson.
-- Third, the route. This is in config/routes.rb, for the forums resource.  As we'll see, this line generates a number of routes.
-- Fourth, the controller.  This is app/controllers/forums_controller.rb.  There are methods in this file to handle each of the actions coming out of the routes.  The methods are very short, meaning that default behavior is used throughout.  However, have a look at the index method.  It starts with:
-    ```
-    @forums = Forum.all
-    ```
-    This invokes the model to get the list of forum records.  This is stored in an instance variable.
-- Finally, let's look at a view, specifically app/views/forums/index.html.erb.  An erb file is an html file with embedded Ruby code, which is always surrounded by:
-    ```
-    <% %>
-    ```
+- First, the migration file.  It is in the `db/migrate` directory.  The name is a timestamp, plus a descriptive name saying what it does.  Open it up.  You'll see that it creates the table in the database.
+- Second, the model.  This is `app/models/forum.rb`.  If you have a look, you'll see that there isn't any code there.  The model gets all of its behavior at this point from the parent class and the schema in the database.  We'll modify model files in a later lesson.
+- Third, the route.  This is in config/routes.rb, for the forums resource.  As we'll see, this line generates a number of routes.
+- Fourth, the controller.  This is `app/controllers/forums_controller.rb`.  There are methods in this file to handle each of the actions coming out of the routes.  The methods are very short, meaning that default behavior is used throughout.  However, have a look at the `index` method.  It starts with:
+
+  ```ruby
+  @forums = Forum.all
+  ```
+
+  This invokes the model to get the list of forum records.  This is stored in the instance variable `@forums`.
+- Finally, let's look at a view, specifically `app/views/forums/index.html.erb`.  An ERB file is an HTML file with embedded Ruby code, which is always surrounded by tags and looks like:
```

Comment:
Added "looks like" because there are also tags like `<%#`, `<%-` and `-%>`, which don't appear on this page.